CHANGES.rst

```diff
@@ -59,6 +59,8 @@ New Features
 
 - ``astropy.wcs``
 
+- Added the injection of EntryPoints into astropy.modeling.fitting if they inherit from Fitters class. [#5241]
```

Comment:
This needs to go into the ``modeling`` section above, and add a line break (probably after ``if`` or ``they``) to match the existing formatting.